packages/opentelemetry-core/src/context/propagation/JaegerHttpTraceFormat.ts

```diff
@@ -38,11 +38,11 @@ export class JaegerHttpTraceFormat implements HttpTextFormat {
     format: string,
     carrier: { [key: string]: unknown }
   ) {
-    const hexTraceId = removeLeadingZeros(spanContext.traceId);
-    const hexSpanId = removeLeadingZeros(spanContext.spanId);
     const flags = TraceFlags.SAMPLED;
```

Comment:
This makes it always sampled. You should get the trace keys from the span context

Reply:
as i get u, do u mean this 
```
    carrier[
      UBER_TRACE_ID_HEADER
    ] = `${spanContext.traceId}:${spanContext.spanId}:0:${spanContext.traceFlags}`;
``` ?

Reply:
spanContext.traceFlags || TraceFlags.UNSAMPLED :)

Reply:
Our span context trace flags are represented as a bitmap, which means it is stored internally as a number. The simplest way to do it is `spanContext.traceFlags.toString(16).padStart(2, "0")` but that is not supported in IE11. Since we only have one flag, I would suggest you just do this:

```typescript
const traceFlags = `0${spanContext.traceFlags.toString(16)}`;
carrier[
      UBER_TRACE_ID_HEADER
    ] = `${spanContext.traceId}:${spanContext.spanId}:0:${traceFlags}`;
```